tensorflow_graphics/rendering/camera/perspective.py

```diff
@@ -266,27 +273,36 @@ def matrix_from_intrinsics(focal, principal_point, name=None):
                                [focal, principal_point]):
     focal = tf.convert_to_tensor(value=focal)
     principal_point = tf.convert_to_tensor(value=principal_point)
+    skew = tf.convert_to_tensor(value=skew)
 
-    shape.check_static(
-        tensor=focal, tensor_name="focal", has_dim_equals=(-1, 2))
+    shape.check_static(tensor=focal, tensor_name="focal", has_dim_equals=(-1, 2))
     shape.check_static(
         tensor=principal_point,
         tensor_name="principal_point",
-        has_dim_equals=(-1, 2))
+        has_dim_equals=(-1, 2),
+    )
+    shape.check_static(
+        tensor=skew,
+        tensor_name="skew",
+        has_dim_equals=(-1, 1),
+    )
     shape.compare_batch_dimensions(
-        tensors=(focal, principal_point),
-        tensor_names=("focal", "principal_point"),
+        tensors=(focal, principal_point, skew),
+        tensor_names=("focal", "principal_point", "skew"),
         last_axes=-2,
-        broadcast_compatible=False)
+        broadcast_compatible=False,
+    )
 
     fx, fy = tf.unstack(focal, axis=-1)
     cx, cy = tf.unstack(principal_point, axis=-1)
     zero = tf.zeros_like(fx)
     one = tf.ones_like(fx)
-    matrix = tf.stack((fx, zero, cx,
-                       zero, fy, cy,
-                       zero, zero, one),
-                      axis=-1)  # pyformat: disable
+
+    skew = tf.reshape(skew, tf.shape(fx))
+
+    matrix = tf.stack(
```

Comment:
There are still unnecessary whitespace changes in this file (for example this matrix). Please remove all unnecessary whitespace changes.